vip_spec.xsd

```diff
@@ -416,13 +404,14 @@ reference elements for polling location
       <xs:element name="BallotSelectionId" type="xs:IDREF" minOccurs="0" maxOccurs="unbounded" />
       <xs:element name="ExternalIdentifiers" type="ExternalIdentifiers" minOccurs="0" />
       <xs:element name="ElectoralDistrictId" type="xs:IDREF" />
-      <xs:element name="Name" type="InternationalizedText" />
-      <xs:element name="Abbreviation" type="InternationalizedText" minOccurs="0" />
+      <xs:element name="Name" type="xs:string" />
+      <xs:element name="Abbreviation" type="xs:string" minOccurs="0" />
       <xs:element name="BallotTitle" type="InternationalizedText" minOccurs="0" />
       <xs:element name="BallotSubTitle" type="InternationalizedText" minOccurs="0" />
       <xs:element name="HasRotation" type="xs:boolean" minOccurs="0" />
       <xs:element name="VoteVariationType" type="VoteVariation" minOccurs="0" />
       <xs:element name="OtherVoteVariationType" type="xs:string" minOccurs="0" />
+      <!-- SURVEY: What is the use case for this field? It's empty in the sample feed. -->
```

Comment:
From the equivalent field in the 3.0 docs: "The electorate_specifications field specifies any changes to the eligible electorate for this contest past the usual, "all registered voters" electorate. This subtag will most often be used for primaries and local elections. In primaries, voters may have to be registered as a specific party to vote, or there may be special rules for which ballot a voter can pull. In some local elections, non-citizens can vote."

e.g. "registered members of Party X only" for primaries.